twitterpuller/src/main/java/de/brandwatch/minianalytics/twitterpuller/TwitterPullerApplication.java

```diff
@@ -24,7 +33,19 @@ public TwitterStream getTwitterStream(TwitterPullerStatusListener listener) {
     }
 
     @Bean(initMethod = "sendTweetsToKafka")
-    public TweetProducer tweetProducer(TwitterStream twitterStream){
+    public TweetProducer tweetProducer(TwitterStream twitterStream) {
         return new TweetProducer(twitterStream);
     }
+
+    @Bean
+    public CacheManager cacheManager() {
+        return new ConcurrentMapCacheManager("{queryID, query}");
+    }
+
+    //Refreh Cache every 10 minutes
+    @CacheEvict(allEntries = true, cacheNames = "{queryID, query}")
```

Comment:
Interesting, haven't used this yet. Can you send me a link to the resource/tutorial you used?

Reply:
https://www.baeldung.com/spring-boot-evict-cache

This one...I think. I don't know 100% anymore. But the things I use are described there.